sys/include/checksum/crc16_ccitt.h

```diff
@@ -37,8 +34,92 @@
 extern "C" {
 #endif
 
+/**
+ * @brief           Update CRC16-CCITT-KERMIT
+ *
+ * @param[in]  crc  A start value for the CRC calculation, usually the
+ *                  return value of a previous call to
+ *                  crc16_ccitt_kermit_calc(), crc16_ccitt_mcrf4xx_calc() or crc16_ccitt_kermit_update()
+ * @param[in]  buf  Start of the memory area to checksum
+ * @param[in]  len  Number of bytes to checksum
+ *
+ * @return          Checksum of the specified memory area based on the
+ *                  given start value
+ */
+uint16_t crc16_ccitt_kermit_update(uint16_t crc, const unsigned char *buf, size_t len);
+
+/**
+ * @brief           Calculate CRC16-CCITT-KERMIT
+ *                  polynom=0x1021
+ *                  seed=0x0000
+ *                  check=0x2189
```

Comment:
Be aware, that the `@brief` should only be one line. Maybe it is best to put the polynom, seed, and check into the detail section. E.g.

```suggestion
 *
 * ~~~~~~~~~~~~~~~~~~~~ {.c}
 * polynom=0x1021
 * seed=0x0000
 * check=0x2189
 * ~~~~~~~~~~~~~~~~~~~~ 
```

(this makes it also rendered as code in doxygen)